### ISSUE TITLE
Sherlock Holmes: The Devil's Daughter (350640)

### ISSUE THREAD

**Issue by neuromancer** [ValveSoftware/Proton#2101]
# Compatibility Report
- Name of the game with compatibility issues: Sherlock Holmes: The Devil's Daughter
- Steam AppID of the game: 350640

## System Information
- GPU: GeForce GTX 1060 6GB
- Driver/LLVM version: NVIDIA 415.22
- Kernel version: 4.19.4
- Proton version: 3.16

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-350640.log](https://github.com/ValveSoftware/Proton/files/2683678/steam-350640.log)

## Symptoms 

The game does not start. 

## Reproduction

Just try to start the game.

**Comment by Guy1524** [comment]
`
8476.423:0024:0025:warn:module:load_dll Failed to load module L"gameoverlayrenderer64.dll"; status=c0000135
8476.454:0024:0025:trace:module:LdrShutdownProcess ()`

It looks like the process shuts down after it is unable to load gameoverlayrenderer64, could this be the cause?

**Comment by neuromancer** [comment]
I don't think so. I believe this game has Denuvo protection and it's failing like [Just Cause 3](https://github.com/ValveSoftware/Proton/issues/682).

**Comment by MegWATTT** [comment]
The game does run with Steam running on Wine. Tested with both tkg-protonifed-faudio-4.5 and tkg-protonified-3.21.

I suspect a problem specific to Proton.

**EDIT:** Looks like only Wine staging is able to run the game. It isn't not possible on Wine devel

**Comment by neuromancer** [comment]
Can we try to isolate the patch that wine-stagging has to be able to play this game?

**Comment by MegWATTT** [comment]
The game does run after applying ntdll patches to wine devel (tested on 4.7). 
I haven't tested with proton.
I applied these patches with patchinstall.sh: 

`ntdll-Activation_Context ntdll-APC_Performance ntdll-ApiSetMap ntdll-Builtin_Prot ntdll-CriticalSection ntdll-Dealloc_Thread_Stack ntdll-DeviceType_Systemroot ntdll-DOS_Attributes ntdll-Exception ntdll-FileDispositionInformation ntdll-FileFsFullSizeInformation ntdll-FileFsVolumeInformation ntdll-Fix_Alignment ntdll-HashLinks ntdll-Heap_Improvements ntdll-Hide_Wine_Exports ntdll-Interrupt-0x2e ntdll-LDR_MODULE ntdll-Manifest_Range ntdll-MemoryWorkingSetExInformation ntdll-NtAccessCheck ntdll-NtContinue ntdll-NtDevicePath ntdll-NtQueryEaFile ntdll-NtQuerySection ntdll-NtQueryVirtualMemory ntdll-NtSetLdtEntries ntdll-Pipe_SpecialCharacters ntdll-ProcessQuotaLimits ntdll-RtlCaptureStackBackTrace ntdll-RtlCreateUserThread ntdll-RtlQueryPackageIdentity ntdll-Serial_Port_Detection ntdll-set_full_cpu_context ntdll-Signal_Handler ntdll-Stack_Guard_Page ntdll-Stack_Overflow ntdll-Status_Mapping ntdll-SystemExtendedProcessInformation ntdll-SystemInterruptInformation ntdll-SystemModuleInformation ntdll-SystemRoot_Symlink ntdll-Threading ntdll-ThreadTime ntdll-User_Shared_Data ntdll-Wait_User_APC ntdll-WRITECOPY ntdll-Zero_mod_name
`

**Comment by neuromancer** [comment]
Neat!, @MegWATTT can you test Just Cause 3 as well? It is more likely that the patches are accepted in Proton if they fix more than one game :smile:

**Comment by MegWATTT** [comment]
I don't have Just Cause 3 unfortunately.
As for Proton, some ntdll patches from Wine staging 4.2 won't apply to Proton 4.2, so I can't test if It works on Proton.

**Comment by neuromancer** [comment]
is this working with Proton 4.11?

**Comment by MegWATTT** [comment]
Unfortunately, no

**Comment by MegWATTT** [comment]
I just tried with a custom build of Proton (Proton-4.15-GE-4 ) which have Wine staging patches. It still doesn't work. There might be a thing specific to proton which prevent the game to start. 
There is also a difference. Starting the game in Wine without Wine staging patch lead to the game to be stuck (and have to be killed). However, starting the game with any version of Proton makes the game to close instantly.

**Comment by Kreyren** [comment]
# Compatibility Report
- Name of the game with compatibility issues: **Sherlock Holmes: The Devil's Daughter**
- Steam AppID of the game: **350640**

## System Information
- OS: **Kreyrock Linux Anubis 9999 x86_64**
- GPU: **AMD ATI Radeon HD 7870 GHz Edition**
- Driver/LLVM version: **PITCAIRN, DRM 3.33.0, 5.3.0-3-amd64, LLVM 9.0.0**
- Kernel version: **5.3.0-3-amd64**
- Link to full system information report as [Gist](https://gist.github.com/):
  - [List of installed packages on debian](https://gist.githubusercontent.com/Kreyren/aa9d92fde978f3184f2dd89ecbf18c84/raw/359563ce1caef707adc1c0ae3115a241c587d716/gistfile1.txt)
  - [List of installed packages on gentoo](https://gist.githubusercontent.com/Kreyren/89d68dddfc48cd4da53a78e505674ec4/raw/f68e2f2dc3748cdf9122ff98e49e59e5b299b3b6/gistfile1.txt)
  - [List of installed packages on exherbo](https://gist.githubusercontent.com/Kreyren/c458b76f876c032e4ac333bd610c4989/raw/5c968644d561cd52e6fd99ae75f85b80ddeecbfd/gistfile1.txt)
  - [Steam info](https://gist.githubusercontent.com/Kreyren/386aa0c9aabc56ce7f05aa4fdc5e7647/raw/691b247b6949f7ba58f72211167207b541eb8d1d/gistfile1.txt)
- Proton version: **1578946641 proton-4.11-12**
- Proton log: https://gist.githubusercontent.com/Kreyren/3aa7935a94e99345c8a5c8489f96cedb/raw/60150a1e167665abb5b1e749a38f80ebae4db390/gistfile1.txt

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.



## Symptoms 
Game won't open, expect `pid 215142 != 215141, skipping destruction (fork without exec?)` in logs (which is based on https://github.com/ValveSoftware/Proton/issues/131 unrelated to proton)

## Reproduction
Run the game




## Sidenote
Game (so far) works on platinum on winehq-staging-5.0-rc4

**Comment by MegWATTT** [comment]
The Denuvo issue is gone with proton 5.0. The game seems to be playable now.

**Comment by neuromancer** [comment]
Neat!, I wonder if other games (E.g. Duke Forever) are also playable.

**Comment by RiantAnda** [comment]
# Whitelist Request
- Name of the game to be whitelisted:Sherlock Holmes: The Devil's Daughter
- Steam AppID of the game: 350640

## System Information
- GPU: AMD Radeon RX 5700 XT
- Driver/LLVM version: Mesa 20.0.7 / LLVM 10.0.0
- Distro version: Arch Linux
- Link to full system information report as [Gist](https://gist.github.com/RiantAnda/f16cbb24bf858bbf4e8c228d0de94487)
- Proton version: Proton 5.0-7

## I confirm:
- [X] that pressing the `Play` button in the Steam client is sufficient.
- [] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options) are necessary to run the game.
- [] that no workarounds other than PROTON_* env variables are necessary.

## Issues
- [X] I haven't experienced any issues.
- [ ] There are no issues left open for this game.
- [ ] Although I consider the gaming experience equal to Windows there are remaining issues:

**Comment by kisak-valve** [comment]
Hello, is anyone seeing an issue with this game and Proton 5.0 which would block it from being whitelisted?

**Comment by MegWATTT** [comment]
Hello, the game seems to work on my side without any blocking issues.
The only ones I have is high stuttering when entering new areas, not always related to shader compilation. I've never finished the game so I can't tell if everything inside is working.

**Comment by minisori** [comment]
I finished the game with proton 5.0-9. It's working out of the box very well but as others mentioned, sometimes there is a lot stuttering for a few seconds when moving the camera around, maybe it's due to texture loading.
I also noticed some lighting problems when moving the camera in a few places.
I think both issues are present in windows too.

**Comment by pillowtrucker** [comment]
I have to use proton8 for this, in proton9 my mouse is being kicked out of the window and I can't click on anything. There is a cursor inside of the window drawn by the game, but it's not the real one. I'm using xorg.
